CONTRIBUTING.md

```diff
@@ -124,6 +126,7 @@ The guidelines are changed to:
   message.
 
 Closes: #124
+
```

Comment:
I don't believe this is necessary - both Closes and Signed off can be part of the footer.

Reply:
https://github.com/eclipse/openj9/pull/1636#issuecomment-395081510 seems to imply otherwise. @IBMJimmyk can you validate the correctness of your comment?

Reply:
Actually @DanHeidinga there is a clear example of the check failing in https://github.com/eclipse/openj9/pull/2098. You can see the failure output (https://ci.eclipse.org/openj9/job/PullRequest-SignedOffByCheck-OpenJDK10/244/):

```
13:32:51 ###################################
[Pipeline] echo
13:32:55 FAILURE - The following commits appear to have an incorrect sign-off
[Pipeline] echo
13:33:02 commit afc2c20e928648a6277b2d17f5eebd01b2e75699
13:33:02 Author: Daniel Hong <daniel.hong@live.com>
13:33:02 Date:   Thu May 31 09:55:11 2018 -0400
13:33:02 
13:33:02     Re-enable toLowerCase/toUpperCase acceleration
13:33:02     
13:33:02     Modify API call so that caseConversionHelper recieves
13:33:02     underlying arrays and length of string.
13:33:02     
13:33:02     Closes: #1934
13:33:02     Signed-off-by: Daniel Hong <daniel.hong@live.com>
[Pipeline] echo
13:33:05 ###################################
```

Reply:
Note the PR referenced has since been fixed up but I've linked the failing job in the above comments. It's easily reproducible as well.

Reply:
That just means the check we are implementing hasn't been implemented correctly. Its not the Eclipse ip-validation which was failing.

Reply:
So do we need an issue against our own check to handle this case and close this PR?

Reply:
There already is an issue open, the check is WIP and being testing on the OpenJ9 repo. 
https://github.com/ibmruntimes/openj9-openjdk-jdk9/issues/58#issuecomment-395229037
https://github.com/eclipse/openj9/pull/1941

Yes I think we can close this PR.